packages/lib-classifier/src/store/Subject/Subject.spec.js

```diff
@@ -7,6 +7,7 @@ import { ProjectFactory, SubjectFactory, WorkflowFactory } from '@test/factories
 import stubPanoptesJs from '@test/stubPanoptesJs'
 import RootStore from '../'
 import subjectViewers from '@helpers/subjectViewers'
+import { expect } from 'chai'
```

Comment:
I think this gets added erroneously per VSCode? Might just be me (and my VSCode config) though.

Reply:
Yeah it does. It's autocompleting based on usage.